src/lpython/parser/semantics.h

```diff
@@ -49,6 +49,13 @@ static inline T** vec_cast(const Vec<ast_t*> &x) {
     return s;
 }
 
+static inline stmt_t** IFSTMTS(Allocator &al, ast_t* x)
+{
+    stmt_t **s = al.allocate<stmt_t*>();
+    *s = down_cast<stmt_t>(x);
+    return s;
+}
+
```

Comment:
Yup, this is another way to do it, but we already have a function that does the same job!